File: frontend/webshop/src/app/models/cart-item.ts
import { Review } from "./review";

export class CartItem {
    productId: number;
    name: string;
    brand: string;
    type: string;
    size: number;
    price: number;
    amount: number;
    review: Review;
    imageUrl: string;
    isAvailable?: boolean;

  }
  

File: backend/webshop/order/src/main/java/com/fer/hr/order/model/CartItem.java
package com.fer.hr.order.model;

import com.fer.hr.clients.review.dto.Review;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class CartItem {
    private Long id;
    private Long productId;
    private String name;
    private Double price;
    private Integer amount;
    private Long orderId;
    private Review review;
}
